ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -391,12 +221,7 @@ public override void AddView(BorderedCanvas parent, DependencyObject child, int
         /// <returns>The child view.</returns>
         public override DependencyObject GetChildAt(BorderedCanvas parent, int index)
         {
-            if (HasBorder(parent))
-            {
-                index++;
-            }
-
-            return parent.Children[index];
+            return parent.GetChildAt(index);
```

Comment:
>GetChildAt [](start = 26, length = 10)

Sadly, `UIElementCollection` is sealed. However - you could create a simple wrapper `IList<UIElement>` that exposed more natural methods like `Insert`, `Clear`, etc. Not sure if the additional allocation of the list wrapper would be too expensive.

Reply:
I'm fine with this too though.

---
In reply to: [177606093](https://github.com/Microsoft/react-native-windows/pull/1730#discussion_r177606093) [](ancestors = 177606093)

Reply:
I'll leave it for some future refactoring, especially since I see you talking on some other PRs about doing bigger changes to children management for shadows and such.

---
In reply to: [177606188](https://github.com/Microsoft/react-native-windows/pull/1730#discussion_r177606188) [](ancestors = 177606188,177606093)